lib/rules/indent-function-args.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * @fileoverview Check for function args proper indentation.
+ * @author uberVU
+ * @copyright 2015 uberVU. All rights reserved.
+ * See LICENSE file in root directory for full license.
+ */
+'use strict';
+var getNodeIndent = require('../get-node-indent.js');
+var indentSize = 4;
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = function(context) {
+
+  var allignedWithParen = {
```

Comment:
These [however](https://github.com/uberVU/eslint-plugin-ubervu/pull/2/files#r41531298) should stay outside.